addons/l10n_at/models/template_at.py

```diff
@@ -45,7 +45,8 @@ def _get_at_res_company(self):
     def _setup_utility_bank_accounts(self, template_code, company, template_data):
         super()._setup_utility_bank_accounts(template_code, company, template_data)
         if template_code == "at":
-            company.account_journal_suspense_account_id.tag_ids = self.env.ref('l10n_at.account_tag_external_code_2300')
-            company.account_journal_payment_debit_account_id.tag_ids = self.env.ref('l10n_at.account_tag_external_code_2300')
-            company.account_journal_payment_credit_account_id.tag_ids = self.env.ref('l10n_at.account_tag_external_code_2300')
-            company.transfer_account_id.tag_ids = self.env.ref('l10n_at.account_tag_external_code_2885')
+            bank_tags = self.env.ref('l10n_at.account_tag_external_code_2300') | self.env.ref('l10n_at.account_tag_l10n_at_ABIV')
+            company.account_journal_suspense_account_id.tag_ids = bank_tags
+            company.account_journal_payment_debit_account_id.tag_ids = bank_tags
+            company.account_journal_payment_credit_account_id.tag_ids = bank_tags
```

Comment:
Shouldn't it be += ? :thinking: (or LINK commands)
Just in case there is another override impacting the tags.

Also, do we really need both this and the override in account_journal.py, for liquidity accounts ? :thinking:

Reply:
Done.

But yes, we need that override too (for the bank accounts) as well as this one (for the suspense/outstanding accounts)